index.html

```diff
@@ -24,15 +24,118 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
                 If you like what you see and have a project you need coder, don't 
                 husitate to contact me.
             </p>
-            <h3 class="links">LET'S CONNECT</h3>
+        </main>
+    </section>
+    
+    <!-- First Project -->
+    <section class="work-page">
+        <img src="images/works/tonic/cover1.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
+            <li>Back</li>&nbsp;
+            <li>End</li>&nbsp;
+            <li>Dev</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;
+            <li>2015</li>
+        </ul>
+        <p class="highlights">
+            A daily selection of privately personalized reads
+            no accounts for sign-ups required.
+        </p>
+        <ul class="tags">
+            <li><button>HTML</button></li>
+            <li><button>CSS</button></li>
+            <li><button>Js</button></li>
+        </ul>
+        <button class="project-detail">See Project</button>
+    </section>
+
+    <!-- Second project -->
+    <section class="work-page">
+        <img src="images/works/multi_post_story/cover.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>&nbsp;
+            <img src="images/works/Counter.svg" alt="counter" class="counter">&nbsp;

```

Comment:
- [ ] **OPTIONAL**  :warning: Only li tag is allowed as a child of ul. Please make sure you wrap the img tag inside a li tag